lib/reporting/monthly_idv_report.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+
+require 'csv'
+require 'reporting/identity_verification_report'
+require 'reporting/unknown_progress_bar'
+
+module Reporting
+  class MonthlyIdvReport
```

Comment:
specs for this class?

Reply:
Agreed: this is near the top of the list for tomorrow. :)